tests/Makefile.in

```diff
@@ -514,6 +514,8 @@ test$(EXEEXT): $(TEST_OBJECTS)
 @COND_PLATFORM_MACOSX_1_USE_GUI_1@	
 @COND_PLATFORM_MACOSX_1_USE_GUI_1@	
 @COND_PLATFORM_MACOSX_1_USE_GUI_1@	cp -f $(top_srcdir)/src/osx/carbon/wxmac.icns test_gui.app/Contents/Resources/wxmac.icns
+@COND_PLATFORM_MACOSX_1_USE_GUI_!@	cp -f horse.cur test_gui.app/Contents/Resources/horse.cur
+@COND_PLATFORM_MACOSX_!_USE_GUI_!@	cp -f horse.png test_gui.app/Contents/Resources/horse.png
```

Comment:
How did you manage to mistype this so badly?? `!` should be `1`s here, obviously.